app/mock-google-services.json

```diff
@@ -0,0 +1,51 @@
+{
+  "project_info": {
+    "project_number": "",
+    "project_id": ""
+  },
+  "client": [
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:799842443839:android:c7220d3e3eb8745e630e8a",
+        "android_client_info": {
+          "package_name": "com.prudhvir3ddy.todo_app_gettingthingsdone"
+        }
+      },
+      "oauth_client": [
+        {
+          "client_id": "",
+          "client_type": 3
+        },
+        {
+          "client_id": "",
+          "client_type": 1,
+          "android_info": {
+            "package_name": "com.prudhvir3ddy.todo_app_gettingthingsdone",
+            "certificate_hash": ""
+          }
+        }
+      ],
+      "api_key": [
+        {
+          "current_key": ""
+        }
+      ],
+      "services": {
+        "analytics_service": {
+          "status": 2,
```

Comment:
Do the values of variables like`client type` really matter? i.e., if this value would have been any positive integer other than this specific number(_2_) then, would it have created some difference or this just fits the best in the current scenario/setting of other variables like this?

Reply:
@s-ayush2903 just to pass ci, any random value.